app/config/config.yml

```diff
@@ -1,12 +1,14 @@
 imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
-    - { resource: services.yml }
+    - { resource: ../../src/AppBundle/Resources/config/services.yml }
```

Comment:
register of services in app/config/services.yml it is the good practice http://symfony.com/doc/current/best_practices/business-logic.html#services-naming-and-format